### ISSUE TITLE
pointcloud_to_laserscan nodelet not available in Ubuntu 16.04

### ISSUE THREAD

**Issue by sheldonrb** [ros-perception/pointcloud_to_laserscan#22]
Running >rosrun nodelet declared_nodelets
Does not show pointcloud_to_laserscan in either ROS-kinetic or ROS-lunar. 
System was installed on Ubuntu-16.04 using command 
  >sudo apt-get install ros-kinetic-desktop-full  (or ros-lunar-desktop-full)
If nodelets aren't available, does this require an installation of catkin and a ros launch file
such as: https://answers.ros.org/question/37312/problem-running-pointcloud_to_laserscan-launch-file/ ?

**Comment by paulbovbel** [comment]
I've verified that the nodelet is definitely available in 16.04, assuming the appropriate debian package has been installed. I would try the bringup on a clean PC or docker container to verify your environment.

**Comment by sheldonrb** [comment]
Paul,
 My apologies, I didn't know how to go back and edit my request.
 
In my Ubuntu16.04 release, the package is missing in ros-kinetic-pcl-ros, but present in ros-kinetic-pointcloud-to-laserscan. I didn't realize I needed to add that package to my ROS install, (the documentation for ROS is terrible), but eventually figured it out! Thanks for making it available.
 
 -yours truly,
Rob Sheldon

The Long Ascent: Genesis 1-11 in Science and Myth, vol 1

 
 

-------- Original Message --------Subject: Re: [ros-perception/pointcloud_to_laserscan]pointcloud_to_laserscan nodelet not available in Ubuntu 16.04 (#22)From: Paul Bovbel <notifications@github.com>Date: Mon, December 03, 2018 10:17 amTo: ros-perception/pointcloud_to_laserscan<pointcloud_to_laserscan@noreply.github.com>Cc: sheldonrb <rbs@rbsp.info>, Author <author@noreply.github.com>
I've verified that the nodelet is defintitely available in 16.04. I would try the bringup on a clean PC or docker container to verify your environment.
